src/tss2-esys/esys_crypto_ossl.c

```diff
@@ -22,7 +22,6 @@
 #define LOGMODULE esys_crypto
 #include "util/log.h"
 #include "util/aux_util.h"
-#include "esys_crypto_ossl.h"
```

Comment:
IMHO, a XXX.c file should always include the XXX.h file. But probably as very first...